docs/prompting/zero_shot/style_prompting.md

```diff
@@ -1,7 +1,43 @@
 ---
-title: ""
+title: "Style Prompting"
 description: ""
-keywords: ""
+keywords: "prompt, prompting, prompt engineering, llm, gpt, openai, style prompting, model, AI, python, instructor, zero-shot, role prompting"
 ---
 
-[wip]
+Style prompting<sup><a href="https://arxiv.org/abs/2302.09185">1</a></sup> includes a style, tone, or genre in the prompt. The effect can be similar to [role prompting](https://python.useinstructor.com/prompting/zero_shot/role_prompting/).<sup><a href="https://arxiv.org/abs/2406.06608">\*</a></sup>
+
+```python
+import instructor
+from pydantic import BaseModel
+import openai
+
+class Email(BaseModel):
+    recipient: str
+    sender: str
+    message: str
+
+client = instructor.from_openai(openai.OpenAI())
+
+style_prompt = {
+    "role": "user",
+    "content": (
+        "Write a email addressed to Mr. John Smith from Jane Doe."
+        "The email should be formal and the message should be polite and respectful." # Style prompting
```

Comment:
Again use an admonition if possible, it looks a bit nicer. I think we should use the default format instead of separating the style_prompt. I suggest doing something like

```
write an email addressed to me john smith from jane doe \
<style prompt section> #1 \
... rest of prompt
```
1. ...< description of it>

https://squidfunk.github.io/mkdocs-material/reference/admonitions/